index.js

```diff
@@ -18,39 +19,85 @@ const checkConfigFile = file => {
   }
 }
 
+const readStreamToString = async (inputStream, encoding) => {
+  return new Promise((resolve, reject) => {
+    let chunks = ''
+    inputStream.setEncoding(encoding)
+    inputStream
+      .on('error', reject)
+      .on('data', chunk => (chunks += chunk))
+      .on('end', () => resolve(chunks))
+  })
+}
+
+const createMemoryStream = input => {
+  var stringStream = new stream.Readable()
```

Comment:
✏️ let

Reply:
Yes, of course! I pushed an update.

I'd expect that `standard` would catch code-formatting issues...

(And my original comment got lost here somehow.)